Supplement the kernel's .gitignore

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>
Copyright (C) 2015 david_yang <david_yang@accton.com>

SPDX-License-Identifier:     GPL-2.0

- Ignore platform device tree blob files
- vmlinux.strip

diff --git a/.gitignore b/.gitignore
index 57af07c..3eaa29a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -40,6 +40,7 @@ modules.builtin
 /TAGS
 /linux
 /vmlinux
+/vmlinux.strip
 /vmlinuz
 /System.map
 /Module.markers
diff --git a/arch/arm/boot/.gitignore b/arch/arm/boot/.gitignore
index ce1c5ff..3c79f85 100644
--- a/arch/arm/boot/.gitignore
+++ b/arch/arm/boot/.gitignore
@@ -3,3 +3,4 @@ zImage
 xipImage
 bootpImage
 uImage
+*.dtb
diff --git a/arch/powerpc/boot/.gitignore b/arch/powerpc/boot/.gitignore
index 12da77e..652e35d 100644
--- a/arch/powerpc/boot/.gitignore
+++ b/arch/powerpc/boot/.gitignore
@@ -21,6 +21,7 @@ uImage
 cuImage.*
 dtbImage.*
 treeImage.*
+vmlinux.strip
 zImage
 zImage.initrd
 zImage.bin.*
@@ -44,4 +45,4 @@ fdt_sw.c
 fdt_wip.c
 libfdt.h
 libfdt_internal.h
-
+*.dtb
